src/App.js

```diff
@@ -1,29 +1,15 @@
-import logo from './logo.svg';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+import React, { Component } from 'react';
+import Calculator from './Components/Calculator';
 
-export default App;
+// eslint-disable-next-line react/prefer-stateless-function
+export default class App extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
While you did a nice job working on the project requirements I noticed you disabled eslint it is against microverse policy to disable linters because linters help us write clean code so it would be nice if you enabled eslint.

Reply:
Constructor is added as Linters are not allowed to add a class bases component without adding a constructor.